### DISCUSSION TITLE
What features do you think the dynamic-cli want??

### DISCUSSION THREAD

**Discussion by lainq** [IndianOpenSourceFoundation/dynamic-cli#70] [Ideas]
## What features do you think the dynamic-cli want??

The dynamic-cli has been growing at a faster rate and new features are added to it regularly.

Please help us with your ideas 🙂

**Comment by GouravSardana** [comment]
API testing is the next Goal of Dynamic-CLI :)

**Reply by Souvikns** [reply]
Do by API testing you mean to say kinda like [curl](https://curl.se/), where you basically fetch the data or something like [newman](https://github.com/postmanlabs/newman).

**Reply by GouravSardana** [reply]
Yes. It's kinda curl and we implemented get, post, delete already. Now we will improve it and add auto completion to make a smooth user experience

**Reply by Souvikns** [reply]
I was checking its feature list, and I think it is cool if I can access some sites like StackOverflow directly from my terminal. Do check [ink js](https://github.com/vadimdemedes/ink) if this could make the development feel easy.

**Reply by GouravSardana** [reply]
ink js is really cool. Let me think if we can integrate that with dynamic cli by making a different micro service or something similar.

@pranavbaburaj any suggestions ?

**Comment by GouravSardana** [comment]
Suggestions are always welcome

**Comment by GouravSardana** [comment]
@pranavbaburaj I was thinking to implement (this)[https://github.com/nvbn/thefuck] . Similar suggestion will give dynamic CLI
